report/chapters/introduction.typ

```diff
@@ -3,21 +3,22 @@
 = Introduction
 The project is being carried out in collaboration with KMD, a company located in Aalborg that develops IT solutions to meet the evolving digital needs of modern societies. 
 
-The process of finding a collaborative company began by searching through the company guidebook “Peiling”, which was delivered by attending a student job fair. After finding a few interesting companies, emails were sent to arrange meetings. During the initial meeting with KMD, they expressed significant interest in the collaboration and wants to provide as many resources as possible, such as providing interview participants and access to a hosting server. Their enthusiasm gave motivation and a desire to collaborate with them. 
+The process of finding a collaborative company began by searching through the company guidebook “Peiling”, which was delivered by attending a student job fair. After finding a few interesting companies, emails were sent to arrange meetings. During the initial meeting with KMD, they expressed significant interest in the collaboration and want to provide as many resources as possible, such as providing interview participants and access to a hosting server. However, a restraint given is that it must cost 0 DKK. Their enthusiasm gave motivation and a desire to collaborate with them. 
 
-The aim of this project is to develop a functional application that prioritizes user experience and meets the client’s needs. The primary requirement for this system is to develop a holiday planner to track non-working days for the company’s employees. KMD already has a holiday planner, but since it’s outdated, they want to replace it with an improved version. 
+The aim of this project is to develop a functional application that prioritizes user experience and meets the client’s needs. The primary requirement for this system is to develop a vacation planner to track non-working days for the company’s employees. KMD already has a vacation planner, but since it is outdated, they want to replace it with an improved version. 
```

Comment:
Ikke enig i at it's skal være it is, ser personlig ingen grund til at udvide mere end nødvendigt.